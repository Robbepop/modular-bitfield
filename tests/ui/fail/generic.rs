use modular_bitfield::prelude::*;

#[bitfield]
struct Generic<T> {
    t: core::marker::PhantomData<T>,
}

fn main() {}
