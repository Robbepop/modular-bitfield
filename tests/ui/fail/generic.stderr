error: bitfield structs can only use const generics
 --> tests/ui/fail/generic.rs:4:15
  |
4 | struct Generic<T> {
  |               ^^^
